app/gui/src/dashboard/layouts/AssetsTable.tsx

```diff
@@ -1192,6 +1192,7 @@ export default function AssetsTable(props: AssetsTableProps) {
       }
     }
     // This effect MUST only run when `initialProjectName` is changed.
+    // eslint-disable-next-line react-compiler/react-compiler
```

Comment:
Not sure we should suppress the react-compiler here. AssetsTable is a huge component, and one line shouldn't deopt the whole component.
IMO, there are 2 ways - fix the issue or extract the deopt code into a separate component/hook.

Same goes for other cases